src/types/onyx/HybridApp.ts

```diff
@@ -0,0 +1,10 @@
+/**  State and configuration of a HybridApp */
+type HybridApp = {
+    /** Specifies if the transition from OldDot was made to display a specific subset of screens in NewDot */
+    isSingleNewDotEntry?: boolean;
+
+    /** Indicates if the NewDot is being closed */
```

Comment:
Minor: Indicates if ~the~ NewDot is being closed

Reply:
done 🫡